classroom.rb

```diff
@@ -0,0 +1,18 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student&.classroom = self
+    @students << student
+  end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] Hey there! 🎓 To ensure your project aligns with the Keep It Simple (KISS) principle of Object-Oriented Programming, it's a great idea to move the method that represents the 'belongs-to' relationship (like a student belonging to a classroom) into the Student class within the student.rb file. 📦 This adjustment simplifies the structure, making it more intuitive and easy to manage, following OOP best practices. 🌟👨‍🎓🏫

Reply:
@zeff96 , When I add this method into the student.rb file. 
```
def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
```